keycloak/keycloak-admin-ui
# Users: Adds user details tab and save/update functionality

## Motivation


https://issues.redhat.com/browse/APPDUX-864


## Brief Description

Adds details tab to users section. 

## Verification Steps


1. Go to Users section.
2. Click "Add User".
3. Fill in all fields of the user form.
4. Click "Create".
5. Find the user in the data list and click on the user.
6. Verify that the information in the "Details" tab matches the information you entered.
7. Change some values in the "Details" tab and click "Save".
8. Go back to the data list. 
9. Click on the user again and verify that the changes you made have been updated in the "Details" tab.
10. Run cypress tests.


## Checklist:

ce/output.
